nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -1,15 +1,15 @@
-# Nixos 25.05 (“Warbler”, 2025.05/??) {#sec-release-25.05}
+# NixOS 25.05 (“Warbler”, 2025.05/??) {#sec-release-25.05}
 
 ## Highlights {#sec-release-25.05-highlights}
 
 <!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
 
+In addition to numerous improvements to NixOS modules and general improvemnts, this release has the following highlights:
```

Comment:
This sounds a little off "numerous improvements to NixOS modules and general improvements" (typo here btw). 

Maybe something like:

```
Alongside many enhancements to NixOS modules and general system improvements, this release features the following highlights:
```

Reply:
Thank you!